### PR TITLE
SPCTR-3013 Fix: Marketing Button - Resolved an issue where border width and paddings were not applied correctly in the editor's Tablet/Mobile view.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- https://brainstormforce.atlassian.net/browse/SPCTR-3013

### Screenshots

- https://share.getcloudapp.com/JrueqqRA

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Added to the card

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by aapatre — APPROVED** [review]
The `.has-background` background selector indicates that the fix might be related to some themes, like the Twenty-Twenty-One theme. If yes, then we should also mention about the themes in the changelog.

File: README.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Marketing Button - Resolved an issue where border width and paddings were not applied correctly in the editor's Tablet/Mobile view.
```
Comment (bsf-zanev) [inline README.md:170]:
```suggestion
* Fix: Marketing Button - Resolved an issue wherein the border-width and padding would not work as intended in the editor's tablet & mobile view.
```

**Review by bsf-zanev — APPROVED** [review]

**Comment by bsf-zanev** [comment]
> The `.has-background` background selector indicates that the fix might be related to some themes, like the Twenty-Twenty-One theme. If yes, then we should also mention about the themes in the changelog.

The `:not(.has-background)` psuedo-selector was used in the Desktop Selectors for a while now, this fix is just to ensure that the Desktop priority doesn't override the tablet and mobile in the editor due to the psuedo-selector.

**Review by adi3890 — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (AbhijitNage123) [reply README.md]: Done

**Pull request merged**